lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -30,9 +31,20 @@ export class GalaxyCollectionDatasource extends Datasource {
     packageName,
     registryUrl,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
+    // istanbul ignore if: should never happen because of defaultRegistryUrls
+    if (!registryUrl) {
+      return null;
+    }
+
     const [namespace, projectName] = packageName.split('.');
 
-    const baseUrl = `${registryUrl}api/v2/collections/${namespace}/${projectName}/`;
+    // joinUrlParts (url-join) removes the trailing slash, so adding it back
```

Comment:
Too hard to extend `joinUrlParts()` to accept this option because it is already variable length?

Reply:
Exactly……

Adding `joinUrlPartsWithTrailingSlash` and use it would be an option?

Reply:
I think that would be more elegant